frontend/src/components/board/items/RetroItem.tsx

```diff
@@ -80,10 +82,25 @@ const useStyles = makeStyles((theme) => ({
   likeButton: {
     marginLeft: "auto",
   },
+  discussedBadge: {
```

Comment:
The colors for light theme don't match quite well. It would be nice if we could have here an outlined button. You can toggle the theme on the settings above with the "Change Theme" button. Furthermore, you can use the `card` class as an example, if you need more control regarding the current theme.

Reply:
Good point. I didn't paid much attention to the theme. I though a bit what would be a good implementation from a UI perspective for the tag and decided against a button. A button would suggest an action is happening after it is clicked, which is currently not the case. So I discovered the Clip component from Material UI and I think it is exactly what we need. I chose the outline version as you preferred it for the button.